src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,485 @@
+package duke;
+
+import duke.storage.Storage;
+
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+import duke.ui.Ui;
+
+import duke.dukeException.DukeException;
+
+import duke.parser.Parser;
+
+public class Duke {
+    public static String home = System.getProperty("user.home");
+    public static String filePath = home + "/Desktop/duke.txt";
```

Comment:
i think it would be helpful to leave a line  to separate the static variables with the non-static ones.